### PR TITLE
Added guidelines for reviewers

### PR DESCRIPTION
**Pull request by romanblanco**

### CODE REVIEW

File: reviewers_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Reviewer's guidelines
+
+We have some technical debt that is continously being refactored.
+However, it's possible that someone is getting inspired by wrong pattern that still lays in the codebase.
+In such case the reviewer might not be aware of _The Right Way_ and approves the changes.
+
+This can cause a significant growth in technical dept. As it is complicated to track all the ongoing refactoring approaches,
+this document is intended to summarize common pitfalls and also good examples.
+
+**Please update this document with simple `wrong` and `right` exapmles that you are aware of.**
```
Comment (gmcculloug) [inline reviewers_guidelines.md]: `exapmles` => `examples `

File: reviewers_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+**When doing reviews, please try to go through these exaples to prevent introducing technical debt**
```
Comment (gmcculloug) [inline reviewers_guidelines.md]: `exaples` => `examples`

**Comment by romanblanco** [comment]
@Fryguy, can you review?

**Comment by himdel** [comment]
Cc @martinpovolny, @mzazrivec  we should be adding more stuff to this document..

**Pull request merged**